### PR TITLE
test: Add notes on how to generate data/wallets/high_minversion

### PR DESCRIPTION
**Pull request by maflcko**
I forgot to do this in #16796

### CODE REVIEW

File: test/functional/data/wallets/high_minversion/GENERATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The wallet has been created by starting Bitcoin Core with the options
+`-regtest -datadir=/tmp -walletdir=$(pwd)/test/functional/data/wallets/`.
```
Comment (promag) [inline test/functional/data/wallets/high_minversion/GENERATE.md]: nit, could avoid loading existing wallets by adding `-nowallet` here.

**Review by promag — COMMENTED** [review]
Code review ACK, instructions LGTM.

**Review by fanquake — APPROVED** [review]
ACK fabe4304f484f6da7e7ac3d28921d1ceb7ac93d6

File: test/functional/data/wallets/high_minversion/GENERATE.md (hunk 1 above)
Comment (MarcoFalke) [reply test/functional/data/wallets/high_minversion/GENERATE.md]: TIL

Done

**Review by ryanofsky — APPROVED** [review]
ACK 2222c96deec0f636dee6e49efb745f29b06a40a5

**Pull request merged**